include/openenclave/attestation/verifier.h

```diff
@@ -154,7 +154,11 @@ oe_result_t oe_verifier_free_format_settings(uint8_t* settings);
  * address of the dynamically allocated list of claims (including base and
  * custom).
  * @param[out] claims_length If not NULL, the length of the claims list.
- * @retval OE_OK The function succeeded.
+ * @retval OE_OK The function succeeded, the claim OE_CLAIM_TCB_STATUS should be
+ * "up_to_date" or "software_hardening_needed".
+ * @retval OE_TCB_LEVEL_INVALID The function succeeded, the claim
+ * OE_CLAIM_TCB_STATUS should not be "up_to_date" and
+ * "software_hardening_needed".
```

Comment:
Why strings instead of defines?   Strings are much more susceptible to typos and hence are more bug prone.

Reply:
A string `OE_CLAIM_TCB_STATUS` is easier for user to read their TCB status. But as your point out it could be bug prone, we can passed out the original status in form of uint32_t and let users parse it themselves?

Reply:
Updated.